### DISCUSSION TITLE
[v2] API call with layerid not returning features.

### DISCUSSION THREAD

**Discussion by hbandi** [quay/clair#1175] [Q&A]
Hi Team,

We are using Clair 2.0.8 version of our production environment, we are in the process to upgrade to v4.
But we are facing an issue with the Clair API.
curl -X GET "http://host:6060/v1/layers/{layerId}/feature" this API is returning an error with no data found.
But in our dev environment, we are able to get the data. 
is there anything missing here?

- Clair version/image: 2.0.8
- Clair client name/version: klar
- Host OS: centos
- Kernel (e.g. `uname -a`): 
- Kubernetes version (use `kubectl version`): 
- Network/Firewall setup:

**Comment by ldelossa** [comment]
@hbandi can you post logs?

**Reply by hbandi** [reply]
Not working environment logs
![image](https://user-images.githubusercontent.com/901110/107110693-04239a00-6870-11eb-8e62-18d74cfd452b.png)
working environment logs 
{"Event":"Handled HTTP request","Level":"info","Location":"router.go:57","Time":"2021-02-06 05:59:54.425633","elapsed time":2074043,"method":"GET","remote addr":"127.0.0.1:46296","request uri":"/v1/layers/e2927026e0e7fe3d34c3212922dd086ecd6caf8ed69494c83f6ce62261982d39?features","status":"404"}
{"Event":"Handled HTTP request","Level":"info","Location":"router.go:57","Time":"2021-02-06 06:00:09.619524","elapsed time":7922884,"method":"GET","remote addr":"127.0.0.1:46296","request uri":"/v1/layers/6f6e5c827a55c0d9e1b229311b8101ded665d08c55727586039d4172deffd0a7?features","status":"200"}
{"Event":"Handled HTTP request","Level":"info","Location":"router.go:57","Time":"2021-02-06 06:10:08.609911","elapsed time":6426992,"method":"GET","remote addr":"127.0.0.1:46296","request uri":"/v1/layers/6f6e5c827a55c0d9e1b229311b8101ded665d08c55727586039d4172deffd0a7?features","status":"200"}

**Reply by hbandi** [reply]
Thank you for helping us

**Reply by ldelossa** [reply]
You'll need to do some manual digging to really troubleshoot this issue, we do not have access to your environment. 

If the issue appears to be okay in your dev environment, but not prod, there is most likely an issue with your environments.
